charts/newrelic-k8s-metrics-adapter/templates/apiservice/job-patch/job-patchAPIService.yaml

```diff
@@ -3,50 +3,58 @@ apiVersion: batch/v1
   annotations:
     "helm.sh/hook": post-install,post-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
   labels:
-    app: {{ template "newrelic-k8s-metrics-adapter.name" $ }}
-{{ include "newrelic-k8s-metrics-adapter.labels" $ | indent 4 }}
+    app: {{ template "common.naming.name" $ }}
+{{ include "common.labels" $ | indent 4 }}
 spec:
   template:
     metadata:
-      name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice-patch
+      name: {{ template "common.naming.fullname" . }}-apiservice-patch
       labels:
-        app: {{ template "newrelic-k8s-metrics-adapter.name" $ }}
-{{ include "newrelic-k8s-metrics-adapter.labels" $ | indent 8 }}
+        app: {{ template "common.naming.name" $ }}
+{{ include "common.labels" $ | indent 8 }}
     spec:
       {{- if .Values.apiServicePatchJob.image.pullSecrets }}
       imagePullSecrets:
         {{- toYaml .Values.apiServicePatchJob.image.pullSecrets | nindent 8 }}
       {{- end }}
       containers:
         - name: patch
-          image: {{ .Values.apiServicePatchJob.image.repository }}:{{ .Values.apiServicePatchJob.image.tag }}
+          image: {{ include "common.images.image" ( dict "imageRoot" .Values.apiServicePatchJob.image "context" .) }}
           imagePullPolicy: {{ .Values.apiServicePatchJob.image.pullPolicy }}
           args:
             - patch
             - --namespace={{ .Release.Namespace }}
-            - --secret-name={{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice
+            - --secret-name={{ template "common.naming.fullname" . }}-apiservice
             - --apiservice-name=v1beta1.external.metrics.k8s.io
-          {{- if .Values.apiServicePatchJob.image.volumeMounts }}
+          {{- with .Values.apiServicePatchJob.extraEnv }}
+          env:
+          {{- toYaml . | nindent 10 }}
+          {{- end }}
+          {{- with .Values.apiServicePatchJob.extraEnvFrom }}
+          envFrom:
+          {{- toYaml . | nindent 10 }}
+          {{- end }}
+          {{- with .Values.apiServicePatchJob.extraVolumeMounts }}
           volumeMounts:
-          {{- include "tplvalues.render" ( dict "value" .Values.apiServicePatchJob.image.volumeMounts "context" $ ) | nindent 10 }}
+          {{- toYaml . | nindent 10 }}
           {{- end }}
-      {{- if .Values.apiServicePatchJob.image.volumes }}
+      {{- with .Values.apiServicePatchJob.extraVolumes }}
       volumes:
-      {{- include "tplvalues.render" ( dict "value" .Values.apiServicePatchJob.image.volumes "context" $ ) | nindent 8 }}
+      {{- toYaml . | nindent 6 }}
       {{- end }}
       restartPolicy: OnFailure
-      serviceAccountName: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice
+      serviceAccountName: {{ include "common.serviceAccount.name" . }}-apiservice
       securityContext:
         runAsGroup: 2000
         runAsNonRoot: true
         runAsUser: 2000
```

Comment:
[This PR is still pending](https://github.com/newrelic/helm-charts/pull/725).

Reply:
Same as the one above